### DISCUSSION TITLE
Where do the weight and print time info come from?

### DISCUSSION THREAD

**Discussion by Jeggo65** [UnchartedBull/OctoDash#1372] [Q&A] [answered]
Hello,

on some older files I have the weight and length of the filament and the print time displayed. PrintTimeGenius is installed an seems to be working. FilamentManager is also installed and working.
I use Prusa Slicer 2.3. At the moment, I have no idea where to adjust the settings.

CU

Jens

**Comment by UnchartedBull** [comment, marked as answer]
Print Time is pulled from PTG if installed, otherwise the default from OctoPrint.

Weight is calculated based on the filament length needed for the print + the filament density and diameter (specified in settings). This is just a rough estimate though, the weight of your slicer probably is more accurate.

**Reply by Jeggo65** [reply]
Thanks for the answer!
I had the following problem:
- I saved the settings of OctoPrint on my "old" system (Raspberry Zero).
- Applied OctoPi (0.1.8) and applied all current updates (OS and OctoPrint).
- After this (with no third party plugin) everything worked and the web interface of OctoPrint showed me the size of the model, filament weight and length.
- Made a restore form my saved settings.
- After this, it did not show the model and filament info for new files (old files are OK).
This is how I solved this problem:
- Uninstall all plugins form OctoPrint (with several reboots).
- After this everything worked fine!
- Installed the same plugins one by one (rebooted when necessary).
- Still everything worked and I finally got the data also in OctoDash

So I think something with the dependencies or libraries of the plugins went wrong.
